Jenkinsfile.upload_release

```diff
@@ -74,7 +74,7 @@ node ('macos1'){
     }
 
     stage('Deploy (Android)') {
-      sh ('bundle exec fastlane android release || exit 0')
+      //sh ('bundle exec fastlane android release || exit 0')
```

Comment:
Why not removing it?

Reply:
i believe we will need this sooner or later

Reply:
Makes sense. Could you add a comment detailing that? Something like `TODO: Reenable when stuff is available`

Reply:
it is available actually

Reply:
I mean some comments detailing why it's commented but not removed :)